src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.figures;
+
+public class RightTriangle extends Figure {
+    private int height;
+    private int side;
+
+    public RightTriangle(String color, String name, int height, int side) {
+        super(color, name);
+        this.height = height;
+        this.side = side;
```

Comment:
The `name` parameter is being passed to the superclass constructor. Ensure that this is necessary, as the superclass might already handle the `name` property. If it's redundant, consider removing it.